ansible_base/oauth2_provider/migrations/0001_initial.py

```diff
@@ -46,6 +46,7 @@ class Migration(migrations.Migration):
                 ('algorithm', models.CharField(blank=True, choices=[('', 'No OIDC support'), ('RS256', 'RSA with SHA-2 256'), ('HS256', 'HMAC with SHA-2 256')], default='', max_length=5)),
                 ('post_logout_redirect_uris', models.TextField(blank=True, help_text='Allowed Post Logout URIs list, space separated')),
                 ('redirect_uris', models.TextField(blank=True, help_text='Allowed URIs list, space separated')),
+                ('app_url', models.URLField(blank=True, default=None, null=True, help_text='The URL of the application.')),
```

Comment:
We don't want to modify the initial migration. This will already be applied to existing systems.

Reply:
This should be fixed with my latest forced push.